docs/source/zero_code_change_examples/plot_kmeans_digits.ipynb

```diff
@@ -0,0 +1,327 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# cuml.accel: Zero-Code Change Acceleration with NVIDIA GPUs.\n",
+    "`cuml.accel` is a new tool in cuML which allows you to accelerate [Scikit-Learn](https://scikit-learn.org/1.5/index.html) estimators on NVIDIA GPUs without having to make any changes to your scripts or notebooks. The following notebook is taken directly from the [Scikit-Learn example gallery](https://scikit-learn.org/1.5/auto_examples/cluster/plot_kmeans_digits.html#sphx-glr-auto-examples-cluster-plot-kmeans-digits-py) to demonstrate how unaltered Scikit-Learn code can be accelerated with `cuml.accel`.\n",
+    "\n",
+    "As always, please remember to [cite](https://scikit-learn.org/1.5/about.html#citing-scikit-learn) Scikit-Learn in any publication based on the fantastic work of the Scikit-Learn community."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The following magic is the only change required to enable GPU acceleration with cuml.accel\n",
+    "%load_ext cuml.experimental.accel\n",
+    "# If you wish to see results WITHOUT cuml.accel, be sure to comment out the above AND restart the notebook kernel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# A demo of K-Means clustering on the handwritten digits data\n",
+    "\n",
+    "In this example we compare the various initialization strategies for K-means in\n",
+    "terms of runtime and quality of the results.\n",
+    "\n",
+    "As the ground truth is known here, we also apply different cluster quality\n",
+    "metrics to judge the goodness of fit of the cluster labels to the ground truth.\n",
+    "\n",
+    "Cluster quality metrics evaluated (see `clustering_evaluation` for\n",
+    "definitions and discussions of the metrics):\n",
+    "\n",
+    "=========== ========================================================\n",
+    "Shorthand    full name\n",
+    "=========== ========================================================\n",
+    "homo         homogeneity score\n",
+    "compl        completeness score\n",
+    "v-meas       V measure\n",
+    "ARI          adjusted Rand index\n",
+    "AMI          adjusted mutual information\n",
+    "silhouette   silhouette coefficient\n",
+    "=========== ========================================================\n"
```

Comment:
It looks like the table is not shown as expected.

Reply:
This is a direct copy from the existing example, and we're looking to move it over with as few changes as possible, so I think this will be okay for now.